faq.md

```diff
@@ -233,10 +232,10 @@ The primary criteria we used to pick a programming language were the following:
   performance and delivers predictable performance, without periodic
   pauses that would cause dropped frames.
 
-* _Fast allocation_. The Flutter framework uses a functional-reactive style
+* _Fast allocation_. The Flutter framework uses a reactive style
```

Comment:
It's actually the functional part that's more relevant here, not the reactive part. The reactive programming model doesn't necessarily imply much memory allocation. What's relevant here is that we use a functional-style flow where many immutable objects get allocated to describe the scene, then the previous scene's objects get released.